app/views/events/show.html.erb

```diff
@@ -82,10 +82,27 @@
 <div class="table-responsive">
   <h3><%= t("events.show.participants") %></h3>
   <table class="table">
-    <thead><tr><th><%=t '.title', default: @event.participant_model.model_name.human.titleize %></th></tr></thead>
+    <thead>
+    <tr>
+      <th><%=t '.title', default: @event.participant_model.model_name.human.titleize %></th>
+      <% if @event.is_a? Rankinglist %>
+        <th><%= @event.human_game_mode %></th>
+      <% end %>
+    </tr>
+    </thead>
     <tbody>
-    <% @event.teams.each do |participant| %>
-      <tr><td><%= link_to_participant participant %></td></tr>
+    <% @event.participants.order(rating: :desc).each do |participant| %>
+      <tr><td><%= link_to_participant participant.team %></td>
+        <% if @event.is_a? Rankinglist %>
+          <td><%= participant.rating %></td>
+          <% if can? :leave, @event %>
```

Comment:
is leave the correct action to test whether a match can be added or not?

Reply:
this is implemented in #272